nbgrader/coursedir.py

```diff
@@ -142,6 +142,18 @@ def _validate_notebook_id(self, proposal: Bunch) -> str:
         )
     ).tag(config=True)
 
+    instantiated_directory = Unicode(
+        'instantiated',
```

Comment:
Maybe `source_with_tests` would be clearer than `instantiated`?

Reply:
Renmaed to `source_with_tests`